docker/publish.sh

```diff
@@ -0,0 +1,24 @@
+set -e
+
+case $1 in
+    base) echo "Publishing '${1}' image" ;;
+    *) 
+        echo "${command} not found" 
+        exit 1;;
+esac
+
+cp $1 Dockerfile
+cp -r ../bash ./bash
+cp -r ../python ./python
+
+DOCKER_IMAGE_NAME=nycplanning/$1
+
+echo "$DOCKER_PASSWORD" | docker login -u $DOCKER_USER --password-stdin
+# Build image - Once we reach some sort of MVP, maybe worth starting versioning. For now, just latest
+docker build \
+    --tag $DOCKER_IMAGE_NAME:latest .
+docker push $DOCKER_IMAGE_NAME:latest
+
+rm Dockerfile
+rm -rf bash
+rm -rf python
```

Comment:
probably doesn't affect anything, but are these needed? maybe especially when it's run locally?

Reply:
They're really just temporary parts of the build, I figure leave the workspace like I found it. The reason for not callin the Dockerfile "Dockerfile" was that we might for now want a few dockerfiles in there, and since there's not a lot of overhead just having the files at the same folder level rather than subfolders made sense to me

Reply:
Though I see your point if this is mainly for running in an action